utils/config.env

```diff
@@ -57,7 +57,7 @@ JAR_VERSION=5.5.0-SNAPSHOT
 
 # TAG - image tag.
 # The ':' which separates the image name from the TAG is not required here
-TAG=5.5.x-latest
+TAG=$CONFLUENT_DOCKER_TAG
```

Comment:
Same comment as above.  This variable was created before the discussion with the automation tools and should be removed.